### ISSUE TITLE
XCode project errors with default config

### ISSUE THREAD

**Issue by gtebbutt** [martiliones/icon-set-creator#19]
I've been having XCode build issues when creating icons using the default configuration - specifically, the asset catalog name is set to the default icon name (`Icon-App`), but then cannot be found by the build process. 

As far as I can see `changeIosLauncherIcon` should take a `catalogName`, not an `iconName`, in order for XCode to find it correctly.

**Comment by martiliones** [comment]
You can specify catalog name with `ios` option. `changeIosLauncherIcon` is actually called with `catalogName`, but the icons are called by the same name, so in `changeIosLauncherIcon` the catalog name argument is called `iconName`.

Example using cli:
```
npx icon-set-creator create --ios YourCatalogName ./assets/icon.png
```

or using `.iconsetrc.js`:
```js
module.exports = {
  imagePath: './assets/icon.png',
  ios: 'YourCatalogName'
};
```

> ⚠️ These examples create icons **only** for IOS. To generate the icons for Android you need to pass `android` option

**Comment by martiliones** [comment]
Feel free to reopen this issue if you have any problems

**Comment by martiliones** [comment]
Feel free to reopen this issue if you have any problems

**Comment by gtebbutt** [comment]
Hi, thanks for the follow up! 

First thing I can confirm is that it wasn't working for me as it stands (default catalog name, no specific parameter), but swapping `iconName` for `catalogName` on the following line totally fixes it: https://github.com/martiliones/icon-set-creator/blob/e98afc955cdcb37ee38878476b367a499d6d8e64/lib/creator/ios.ts#L70

Looking a little further up, if I'm understanding correctly they're both set to `AppIcon` initially, and set to the same value if `ios === 'string'`, but if `flavor` is set or if no parameter is passed then `iconName` will be set to `iosDefaultIconName` and `catalogName` will be set to a different value: 
https://github.com/martiliones/icon-set-creator/blob/e98afc955cdcb37ee38878476b367a499d6d8e64/lib/creator/ios.ts#L44-L57

**Comment by martiliones** [comment]
Thanks for sharing the details! The latest commit should fix this. Could you test it before I push the new version to npm?

**Comment by gtebbutt** [comment]
Hey, I've tried pointing npm at the latest commit but it doesn't seem to run directly from the repo - I'm guessing it's typescript related, but unfortunately that's not something I use in my normal workflow so it's quite possible I'm missing a standard part of the toolchain there!

**Comment by martiliones** [comment]
@gtebbutt I built and published `icon-set-creator@1.1.3`

**Comment by gtebbutt** [comment]
Working perfectly, thanks so much for the fix!
